ghost/core/core/server/api/endpoints/tiers.js

```diff
@@ -47,26 +29,11 @@ module.exports = {
         data: [
             'id'
         ],
-        validation: {
```

Comment:
GitHub won't let me comment on the above, but we can delete the following now too:

```
options: [ 
    'include'
],
```